lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -933,6 +1060,48 @@ impl From<CGOFileSystemObject> for FileSystemObject {
     }
 }
 
+#[derive(Debug)]
```

Comment:
For a protocol this complex, I find doc comments super helpful.

Something as simple as 

```
/// SharedDirectoryCreateRequest is sent from the X to the Y in order to Z.
```

Reply:
Next thing on my todo list is to go through and do this here and for subsequent PRs since it will make them easier to review.